File: bank-service/src/main/java/rs/edu/raf/repository/accounts/ForeignCurrencyBankAccountRepository.java
package rs.edu.raf.repository.accounts;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import rs.edu.raf.model.accounts.ForeignCurrencyBankAccount;

@Repository
public interface ForeignCurrencyBankAccountRepository extends JpaRepository<ForeignCurrencyBankAccount,Long> {

}

File: bank-service-kotlin/src/main/kotlin/org/example/bankservicekotlin/repository/accounts/ForeignCurrencyBankAccountRepository.kt
package org.example.bankservicekotlin.repository.accounts

import org.example.bankservicekotlin.model.accounts.ForeignCurrencyBankAccount
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ForeignCurrencyBankAccountRepository:JpaRepository<ForeignCurrencyBankAccount, Long> {
}